lib/index.js

```diff
@@ -315,7 +325,9 @@ export default {
 			if (!this.console) {
 				this.console = new Console(this.serializedState.console);
 			}
-
+			atom.config.onDidChange('appcelerator-titanium.general.showToolbar', (value) => {
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/index.js line...
  
  
  DangerID: danger-id-default;
  File: lib/index.js;
  Line: 328;
-->

- :warning: lib/index.js line 328 – 'value' is defined but never used. Allowed unused args must match /^_.+/. (no-unused-vars)